code/modules/power/floodlight.dm

```diff
@@ -106,12 +106,9 @@
 	..()
 
 /obj/machinery/power/floodlight/obj_break(damage_flag)
-	if(!(flags_1 & NODECONSTRUCT_1))
-		playsound(loc, 'sound/effects/glassbr3.ogg', 100, 1)
-		var/obj/structure/floodlight_frame/F = new(loc)
```

Comment:
where did all this go? does the parent handle it?

Reply:
Botched the port, fixed now